docs/src/main/paradox/running-the-application.md

```diff
@@ -0,0 +1,86 @@
+Running the application
+-----------------------
+
+You can run the Hello World application from the command line or an IDE. The final topic in this guide describes how to run it from IntelliJ IDEA. However, before we get there, let’s take a quick look at the build tool: sbt.
+
+## The build files
+
+sbt uses a build.sbt file to handle the project. This project’s build.sbt file looks like this:
+
+@@snip [build.sbt]($g8root$/build.sbt)
+
+## Running the project
+
+We run the application from a console/terminal window:
+
+1. Enter `./sbt` on OSX/Linux or `sbt.bat` on Windows
+
+sbt downloads project dependencies. The `>` prompt indicates sbt has started in interactive mode.
+
+2. At the sbt prompt, enter `run`.
+
+The output should look like this:
+
+```
+...
+[info] Running com.lightbend.akka.http.sample.QuickstartServer
+Server online at http://localhost:8080/
+Press RETURN to stop...
+```
+
+## Testing the application
+
+The Akka HTTP server is now running, and we will use the [cURL](https://en.wikipedia.org/wiki/CURL) command to test the application. If you  prefer to use your browser to test the service then a tool like [RESTClient](http://restclient.net/) may be good to install.
```

Comment:
yes, that's true, as it helps you to avoid a lot of boilerplate, compare:

```bash
curl -H "Content-type: application/json" -X POST -d '{"name": "MrX", "age": 31, "countryOfResidence": "Canada"}' http://localhost:8080/user
```

with:

```bash
http http://localhost:8080/user name=MrX age=31 countryOfResidence=Canada 
```

The only downside which I can think of, is that it's not as ubiquitous as good, ol' curl.